import io.netty.util.concurrent.DefaultPromise.StacklessCancellationException;
public class FunctionUse22938 {
public void funcUse() {
StacklessCancellationException stacklesscancellationexception = new StacklessCancellationException();
stacklesscancellationexception.newInstance(null,null);
}
}